woodwork/tests/accessor/test_statistics.py

```diff
@@ -763,6 +763,20 @@ def test_describe_with_include(sample_df):
     multi_params_df["full_name"].equals(sample_df.ww.describe()["full_name"])
 
 
+def test_describe_numeric_all_nans():
+    df = pd.DataFrame({"float": [np.nan] * 5})
+    df.ww.init(logical_types={"float": "double"})
```

Comment:
@thehomebrewnerd sorry about that. Was wondering if it would make sense to also test with IntegerNullable here

Reply:
Should be simple to add. I will update!